tools/jsrun.py

```diff
@@ -31,6 +31,11 @@ def timeout_run(proc, timeout=None, note='unnamed process', full_output=False, n
 
 
 def make_command(filename, engine=None, args=[]):
+  # if no engine is needed, indicated by None, then there is a native executable
+  # provided which we can just run
+  if engine[0] is None:
+    executable = shared.unsuffixed(os.path.abspath(filename)) + '.exe'
+    return [executable] + args
```

Comment:
Not sure about the use of `.exe` here on non-windows platforms.    

Also seems strange to even use "jsrun" to run these native binaries...   I guess there are some larger refactoriings that should happen here.  I've been trying come up with a better way to factor this stuff for a while, but haven't managed to get anything to stick yet.

Reply:
Yeah, i'm not super-happy with `.exe` either, but it existed before (just moving it over here) and it seems unambiguous at least.